discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -54,6 +54,7 @@ def _update_day2_config(self, api_client: InventoryClient, cluster_id: str):
                 high_availability_mode=day2_cluster.high_availability_mode,
                 olm_operators=day2_cluster.monitored_operators,
                 base_dns_domain=day2_cluster.base_dns_domain,
+                hyperthreading=day2_cluster.hyperthreading,
```

Comment:
this is meaningless, since day2 worker receives its configuration from the installed OCP cluster.

Reply:
FIXED